src/streamlit_extras/customize_running/__init__.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+import streamlit as st
+
+center_css = """
+<style>
+
+div[class='css-4z1n4l ehezqtx5']{
```

Comment:
So, I am always too nervous to use random-looking class names like this, as I feel like they might be liable to change at any time. Do you have any idea if these are likely to stay the same, long-term? Happy to ask internally if you don't know, but didn't know if you'd already asked.

Reply:
Yea I wanted to give a double check. Found a cleaner class here https://github.com/arnaudmiribel/streamlit-extras/pull/36/commits/633c6477d1ff553a6fb8f923d92f6b9e0e722778#diff-038353bf37afcfd34ef32ecc32752fdc0b975e6ff65471b8299dbceda013587dR8